src/components/GameItem.js

```diff
@@ -9,9 +9,25 @@ class GameItem extends Component {
     layer: PropTypes.number.isRequired,
   }
 
+  constructor(props) {
+    super(props);
+    this.state = {
+    }
+  };
+
+  
   onItemClicked = () => {
-    // Fill this in for Wave 2!
-  }
+    if (this.props.type === "litter") {
```

Comment:
If this check included a check for itemDesign state, it wouldn't call the increaseScoreCallback after it'd already been spotted.